kanban-all/kanban-data/src/main/java/ru/ifmo/p3411/data/TaskUser.java

```diff
@@ -0,0 +1,48 @@
+package ru.ifmo.p3411.data;
+
+import javax.persistence.*;
+
+/**
+ * Created by Ellepsis on 12.02.2017.
+ */
+@Entity
+@Table(name = "task_user")
+public class TaskUser {
```

Comment:
I suppose these names are crazy, but while I'm not seeing these in work I can tell the better :).